packages/playwright-ct-core/src/tsxTransform.ts

```diff
@@ -14,10 +14,12 @@
  * limitations under the License.
  */
 
-import path from 'path';
-import type { T, BabelAPI, PluginObj } from 'playwright/src/transform/babelBundle';
-import { types, declare, traverse } from 'playwright/lib/transform/babelBundle';
+import * as path from 'path';
```

Comment:
Ditto

Reply:
> Lets try to enable esModuleInterop? Like we have in the tsconfig

I'm planning on doing that, but it is a project in itself.